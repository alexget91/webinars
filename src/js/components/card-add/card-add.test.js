import React from "react";
import renderer from "react-test-renderer";
import CardAdd from "./card-add";

it(`Card add correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<CardAdd onSubmit={jest.fn}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
