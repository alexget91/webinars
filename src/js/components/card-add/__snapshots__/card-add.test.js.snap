// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card add correctly renders after relaunch 1`] = `
<div
  className="card-add"
>
  <form
    className="form"
    onSubmit={[Function]}
  >
    <section
      className="image-upload card-add__image-upload "
    >
      <aside
        className="image-upload__preview-container"
      >
        <button
          className="image-upload__clear"
          onClick={[Function]}
          title="Clear"
        />
      </aside>
      <div
        className="image-upload__drop-zone"
        onBlur={[Function]}
        onClick={[Function]}
        onDragEnter={[Function]}
        onDragLeave={[Function]}
        onDragOver={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        tabIndex={0}
      >
        <input
          accept="image/jpeg, image/png"
          autoComplete="off"
          multiple={false}
          onChange={[Function]}
          onClick={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
          tabIndex={-1}
          type="file"
        />
        <div
          className="image-upload__text-container"
        >
          <p
            className="image-upload__text"
          >
            select an image file to upload or drag it here
          </p>
        </div>
      </div>
    </section>
    <div
      className="form__field"
    >
      <label
        className="form__label"
        htmlFor="card-add-title"
      >
        Title
      </label>
      <input
        className="form__text-input"
        id="card-add-title"
        name="title"
        onChange={[Function]}
        placeholder="Enter title"
        type="text"
      />
    </div>
    <div
      className="form__field"
    >
      <label
        className="form__label"
        htmlFor="card-add-description"
      >
        Description
      </label>
      <textarea
        className="form__text-input card-add__description"
        id="card-add-description"
        name="description"
        onChange={[Function]}
        placeholder="Enter description"
      />
    </div>
    <div
      className="card-add__buttons"
    >
      <input
        className="btn btn--green card-add__submit"
        disabled={true}
        type="submit"
        value="Save"
      />
    </div>
  </form>
</div>
`;
